docs/tutorials/wiki/authorization.rst

```diff
@@ -61,7 +81,20 @@ request)`` returns ``None``.  We will use ``groupfinder()`` as an
 :term:`authentication policy` "callback" that will provide the
 :term:`principal` or principals for a user.
 
-In a production system, user and group data will most often come from a
+There are two helper methods that will help us later when loging-in users.
```

Comment:
`s/loging-in/authenticating` perhaps? I think loging-in is quite difficult to parse if it is even a real word.